PR-App/PR-App/Supporting Files/SceneDelegate.swift

```diff
@@ -18,8 +18,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         
         window = UIWindow(frame: windowScene.coordinateSpace.bounds)
         window?.windowScene = windowScene
-        window?.rootViewController = UINavigationController(rootViewController: LoginScreenVC())
-//        window?.rootViewController = TabBarController()
+//        window?.rootViewController = UINavigationController(rootViewController: LoginScreenVC())
+        window?.rootViewController = TabBarController()
+//        window?.rootViewController = UINavigationController(rootViewController: NewMedicationVC())
+//        window?.rootViewController = UINavigationController(rootViewController: UserMedicationDetailVC())
```

Comment:
Just like mentioned, You could alway stash and delete commented lines :)

Reply:
Yup, we have talked about this, you are right but trying to check each VC is better to uncomment or comment proper root :) But YES you are right :)